code/go/0chain.net/blobbercore/automigration/automigration.go

```diff
@@ -129,20 +134,9 @@ func grantPrivileges(db *gorm.DB) error {
 }
 
 func migrateSchema(db *gorm.DB) error {
-	// !FIXME : tables can't be dropped as default on blobber startup. Because blobber will lose all data if it restarts. It might be a critical bug on production.
-	tablesToKeep := make(map[string]struct{})
-
-	for _, tb := range config.Configuration.DBTablesToKeep {
-		tablesToKeep[tb] = struct{}{}
-	}
-
 	var migratingTables []tableNameI
 	for _, tblMdl := range tableModels {
 		tableName := tblMdl.TableName()
-		if _, ok := tablesToKeep[tableName]; ok {
-			continue
-		}
-
 		err := db.Migrator().DropTable(tableName)
```

Comment:
We need `Migration`. `drop` and `create` is `recreate` with fresh data, is not `schema migration`.